hexapod_dart/include/hexapod_dart/hexapod_dart_simu.hpp

```diff
@@ -198,32 +198,32 @@ namespace hexapod_dart {
             (*d).get(result);
         }
 
-        double covered_distance()
+        double covered_distance() const
         {
             return _covered_distance;
         }
 
-        double energy()
+        double energy() const
         {
             return _energy;
         }
 
-        double arrival_angle()
+        double arrival_angle() const
         {
             return _arrival_angle;
         }
 
-        Eigen::Vector3d final_pos()
+        Eigen::Vector3d final_pos() const
         {
             return _final_pos;
         }
 
-        Eigen::Vector3d final_rot()
+        Eigen::Vector3d final_rot() const
```

Comment:
Do you think it should be worth returning a const reference for the Vector3d methods ? Depends on the cost of the copy constructor.